orchestra/project_api/serializers.py

```diff
@@ -159,10 +159,10 @@ def get_in_progress_task_data(self, obj):
 
 class TimeEntrySerializer(serializers.ModelSerializer):
 
-    worker = serializers.ReadOnlyField(source='worker.id')
-
     class Meta:
         model = TimeEntry
+        fields = ('id', 'date', 'time_worked', 'description', 'assignment')
```

Comment:
cleaned up the serializer so we don't allow updating of fields that shouldn't be updated.